src/torchmetrics/functional/text/bert.py

```diff
@@ -33,6 +35,19 @@
 from torchmetrics.utilities.checks import _SKIP_SLOW_DOCTEST, _try_proceed_with_timeout
 from torchmetrics.utilities.imports import _TQDM_AVAILABLE, _TRANSFORMERS_GREATER_EQUAL_4_4
 
+
+@contextmanager
+def _ignore_log_warning():  # noqa: ANN202
```

Comment:
`mypy` really expects here return type

Reply:
Maybe `contextlib.AbstractContextManager`

Reply:
```suggestion
def _ignore_log_warning() -> Iterator[None]:
```
ref: https://stackoverflow.com/a/49736916/4521646